packages/postcss-plugins-preset/lib/index.js

```diff
@@ -1 +1,7 @@
-module.exports = [ require( 'autoprefixer' )( { grid: true } ) ];
+module.exports = [
+	require( 'autoprefixer' )( { grid: true } ),
+	require( 'postcss-modules' )( {
+		scopeBehaviour: 'global',
+		getJSON: () => {},
+	} ),
```

Comment:
I'm not entirely sure if it's appropriate/necessary to add this to `@wordpress/postcss-plugins-preset`, rather than just for the Gutenberg project. I feel like most WP extenders would not need this, or will have their own setup if they do. It's also non-obvious that `@wordpress/postcss-plugins-preset` would support CSS Modules by default. Thoughts?

There is also a possibility that we will be using CSS Modules for wp-components, and in that case we will need to add additional config (to un-noop the `getJSON`, at the very least) for the Gutenberg project anyway. Not super relevant to the decision we make here, but just adding as an FYI.

Reply:
I agree that we can start with enabling `postcss-modules` only for the Components package :+1:

Reply:
What's the appropriate place to add this config only to Gutenberg?

Reply:
Gutenberg's webpack config in `tools/webpack` perhaps?

Reply:
I think all you need is to change the config passed to `postcss()` here:

https://github.com/WordPress/gutenberg/blob/add/postcss-modules/bin/packages/build-worker.js#L126

It should configure `postcss` with both the preset and the additional plugin.

Reply:
Run `npm run build:packages`. That should build all packages, including styles.

Then look at `packages/edit-site/build-style/style.css`. That's the output of the build, all `edit-site` styles processed and concatenated together.

There is no bundler in this build pipeline, it's just the `bin/packages/build.js` script spawning a farm of `bin/packages/build-worker.js` scripts.